snuba/datasets/table_storage.py

```diff
@@ -0,0 +1,98 @@
+import json
+import rapidjson
+
+from datetime import datetime
+
+from snuba.clickhouse import DATETIME_FORMAT
+from snuba.datasets.schemas.tables import WritableTableSchema
+from snuba.processor import MessageProcessor
+from snuba.writer import BatchWriter
+
+
+class TableWriter:
```

Comment:
I introduced this to avoid the type issues whenI had to set a None processor and a None default topic.
This is a step towards splitting dataset (which represent the datamodel) and the TableStorage which represent the table. 
At this point this contains only what is needed to write on the table but going on it should evolve this way:
1) split Clickhouse schema from snuba schema
2) move Clickhouse schema here
3) rename it to table storage
4) move some single table dataset logic into here
5) keep the snuba schema and the query logic (that runs the join) into the dataset.